src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  film: Movie;
+};
+
+export const MovieCard: React.FC<Props> = (props) => {
+  const { film } = props;
+  const {
```

Comment:
what do you think about using Partial here?

Reply:
If we use Partial here - so we can get an empty object or an object without some important property
And we should write additional conditions for all properties? If it is in object - add some code
I think in this case it's unnecessary)